src/utils/jwt.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable require-jsdoc */
```

Comment:
I generated a jsdoc for this so you can check the changes from my branch

Reply:
I think that they will get merged with mine